source/includes/_bundle.md

```diff
@@ -1454,13 +1474,14 @@ bundle.remove_custom_field(custom_field_id,
 ```
 
 ```python
-bundleApi = killbill.api.BundleApi()
+bundleApi = killbill.BundleApi()
+
 bundle_id = 'ce967207-851c-4040-bfbd-74a8924f9b8a' 
 
 bundleApi.delete_bundle_custom_fields(bundle_id, 
-                                      created_by, 
-                                      api_key, 
-                                      api_secret)
+                                      created_by='demo',
+                                      reason='reason', 
+                                      comment='comment')
```

Comment:
I think the `customFieldId` of the field to be deleted also needs to be specified here. Can you confirm and if so add this parameter? See [this thread.](https://killbillio.slack.com/archives/C02C15Z92PJ/p1692775418598969)

Reply:
Hi @reshmabidikar made the necessary changes.